src/current/_includes/releases/v24.1/v24.1.6.md

```diff
@@ -0,0 +1,87 @@
+## v24.1.6
+
+Release Date: October 17, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-1-6-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Updated the cluster setting [`changefeed.sink_io_workers`]({% link v24.1/cluster-settings.md %}#setting-changefeed-sink-io-workers) with all the [changefeed sinks]({% link v24.1/changefeed-sinks.md %}) that support the setting. [#130373][#130373]
+- Added two network metrics, `changefeed.network.bytes_in` and `changefeed.network.bytes_out`. These metrics track the number of bytes sent by individual [changefeeds]({% link v24.1/change-data-capture-overview.md %}) to the following sinks: 
```

Comment:
@taroface I was taking a look at your PR because I'm working on the 24.2.4 RNs which have a lot of common backports, and noticed that this one needs the PR number 130583 to be re-added.

Reply:
see also [this comment](https://github.com/cockroachdb/docs/pull/19012#discussion_r1804893439)